### DISCUSSION TITLE
In the local environment, the images aren't displayed

### DISCUSSION THREAD

**Discussion by Sergey98Am** [supabase/supabase#21234] [Questions]
For my React app, I use local and remote Supabase projects, to send data from the remote database to the local, I use `npx supabase do dump --data-only -f supabase/seed.sql` command, and after that `npx supabase do reset`.
After running these commands, data was created in a local database with storage, but images aren't displayed in the local environment.

https://github.com/supabase/supabase/assets/52923458/ba72e57c-a2e5-4cbf-abaf-a36f83c7b28c

**Comment by GaryAustin1** [comment]
Moving the storage schema does not move the images in s3.  Normally you should not include the storage object data in your local/versus remote instance transfers.  
https://supabase.com/docs/guides/platform/migrating-and-upgrading-projects#migrate-storage-objects
